pkg/operator/operator.go

```diff
@@ -19,34 +19,28 @@ import (
 
 	operatorapiv1 "github.com/openshift/api/operator/v1"
 	configinformerv1 "github.com/openshift/client-go/config/informers/externalversions/config/v1"
-	proxyvclient1 "github.com/openshift/client-go/config/listers/config/v1"
 	operatorclientv1 "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
 	operatorinformersv1 "github.com/openshift/client-go/operator/informers/externalversions/operator/v1"
+	"github.com/openshift/cluster-openshift-controller-manager-operator/pkg/operator/workload"
```

Comment:
As a follow up to this pull request, it may be worth running our import sorting script on this repo.

Reply:
I think the empty line on the import made the sorting done by my editor different than usual. Let me fix it, and see if it will match expected sequence.